include/sdf/Error.hh

```diff
@@ -147,6 +147,9 @@ namespace sdf
     /// \brief Merge include is unspported for the type of entity being
     /// included, or the custom parser does not support merge includes.
     MERGE_INCLUDE_UNSUPPORTED,
+
+    /// \brief A generic error that is not an sdf error.
+    EXTERNAL_ERROR,
```

Comment:
I was the one who suggested that we add this (https://github.com/ignitionrobotics/sdformat/pull/836#issuecomment-1034079463), but if we're going to assign it a large number, I don't think ABI would be an issue, so perhaps it's better to wait till we need it.

Reply:
ah I see

we can add enum codes without breaking API, so I think it would be fine to omit this for now

Reply:
I removed this type in https://github.com/ignitionrobotics/sdformat/pull/836/commits/851649b4db395da01b34660369269b980b4ef0d6 since we can add it in later if needed; thanks for the feedback :+1: